Calculadora.java

```diff
@@ -0,0 +1,53 @@
+/*
+Escreva um programa em sua linguagem de programação preferida que seja capaz de calcular uma média suicida entre trabalhos e provas.
+
+Teremos um número configurável de quantidade de provas e um número configurável de quantidade de trabalhos.
+
+O cálculo da média se dá:
+
+    Se média das provas >= 5 e média de trabalhos >= 5, a média final será composta por 60% da média de provas e 40% da média de trabalhos
+    Se qualquer uma das médias for menor do que 5, a média final será composta pelo menor média entre as duas médias.
+
+*/
+import java.util.Scanner;
+class Calculadora{
+    static Scanner scan = new Scanner(System.in);
+    static float md_provas = 0;
+    static float md_trabs = 0;
+    static float md_final = 0;
+    
+    static float mdProvas(int qtd_provas) { // metodo para calculo da media das provas
+        for (int prova = 1; prova <= qtd_provas; prova++) {
+            System.out.println("Digite a nota da " + prova + " prova.");
+            md_provas += scan.nextInt(); // somando a nota da prova atual com as anteriores
```

Comment:
```suggestion
            md_provas += scan.nextInt();
```
Esse comentário é meio redundante, não é? Talvez ele não seja necessário. O que acha?